### DISCUSSION TITLE
Fixed labels for y-axis

### DISCUSSION THREAD

**Discussion by MarioMitrevski** [patrykandpatrick/vico#280] [Questions]
Hello, I want to set fixed values for the labels on the y-axis from 0 to 12 with step 1. Can I do this now, and if yes please can you show me how to do that? Thank you in advance

**Comment by patrickmichalik** [comment]
Hello! `VerticalAxis` allows you to specify the maximum label count, which will be used unless there’s not enough room. This enables you to set the step, albeit indirectly. `VerticalAxis` distributes its labels evenly along the axis line.
